lib/engine/game/g_1829/entities.rb

```diff
@@ -221,8 +328,50 @@ module Entities
             type: 'init10',
             color: 'yellow',
             text_color: 'black',
+            abilities: [
+              {
+                type: 'teleport',
+                owner_type: 'player',
+                free_tile_lay: true,
+                hexes: %w[B7 B9 B11 B13 B15 C6 C8 C10 C12 C16 D3 D7 D9 D11 D13 D15 D17 E2 E4 E6 E8 E10 E12 E14 E16 F3 F5 F7 F9 F11
+                          F13 F15 F17 F19 F21 G4 G6 G8 G10 G12 G14 G16 G18 G20 G22 H1 H3 H5 H7 H9 H11 H13 H15 H17 H19 H21 I2 I4
+                          I6 I8 I10 I12 I14 I16 I18 I20 J5 J7 J9 J13 J15 J17 K8 K10 K12 K14 K16 K18 K20 K22 L1 L3 L5 L7 L9 L11
+                          L13 L15 L17 L19 M2 M4 M6 M8],
+                tiles: %w[1a 2a 3a 4a 5 6 7 8 9 10 11s 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33s 34 35
+                          36 37 38 39 40 41 42 43 44 45 46 47 48 49 50s 51 60 59s],
+              },
+            ],
           },
         ].freeze
+
+        PAR_BY_CORPORATION = {
+          'LNWR' => 100,
+          'GWR' => 90,
+          'Mid' => 82,
+          'LSWR' => 76,
+          'GNR' => 71,
+          'LBSC' => 67,
+          'GER' => 64,
+          'GCR' => 61,
+          'LYR' => 58,
+          'SECR' => 56,
+        }.freeze
+
+        REQUIRED_TRAIN = {
+        }.freeze
+
+        # combining is based on http://fwtwr.com/fwtwr/18xx/1829/privates.asp
+        def game_companies
+          comps = []
```

Comment:
def game_companies
  @units[1] ? UNIT1_COMPANIES.dup : []

same for corporations